webpack/scenes/AlternateContentSources/Create/Steps/ACSCredentials.js

```diff
@@ -37,7 +39,18 @@ const ACSCredentials = () => {
         title={__('Credentials')}
         description={__('Enter basic authentication information or choose content credentials if required for this source.')}
       />
+      {verifySSL && acsType === 'rhui' &&
```

Comment:
`verifySSL` doesn't need to be true here. That's for peer validation rather than authentication with the server.  (In fact, the original Pulp KCS says to keep it false -- I'll need to double check that it's even okay for it to be true)

Reply:
> Also, for some reason, in the review details page, the source type is showing up as "Rhui"...

This was just implemented in https://github.com/Katello/katello/pull/10289.